From de2a9bbef23846e203fda70d5b640c0a57633f82 Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Tue, 20 Dec 2016 08:43:30 -0800
Subject: scsi: qedi: fix build, depends on UIO
References: bsc#1004527, FATE#321744
Git-commit: 7961d53d22375bb9e8ae8063533b9059102ed39d
Patch-mainline: v4.10-rc7

Fix build of SCSI qedi driver. It uses uio interfaces so it should
depend on UIO.

Error: "uio_unregister_device" [drivers/scsi/qedi/qedi.ko] undefined!
Error: "uio_event_notify" [drivers/scsi/qedi/qedi.ko] undefined!
Error: "__uio_register_device" [drivers/scsi/qedi/qedi.ko] undefined!

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Cc: QLogic-Storage-Upstream@cavium.com
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Johannes Thumshirn <jthumshirn@suse.de>

---
 drivers/scsi/qedi/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/scsi/qedi/Kconfig b/drivers/scsi/qedi/Kconfig
index 23ca8a2..2133145 100644
--- a/drivers/scsi/qedi/Kconfig
+++ b/drivers/scsi/qedi/Kconfig
@@ -1,6 +1,6 @@
 config QEDI
 	tristate "QLogic QEDI 25/40/100Gb iSCSI Initiator Driver Support"
-	depends on PCI && SCSI
+	depends on PCI && SCSI && UIO
 	depends on QED
 	select SCSI_ISCSI_ATTRS
 	select QED_LL2
-- 
1.8.5.6

